test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -289,70 +288,85 @@ public void PlattCalibratorOnnxConversionTest()
 
             var initialPipeline = mlContext.Transforms.ReplaceMissingValues("Features").
                 Append(mlContext.Transforms.NormalizeMinMax("Features"));
+            return (mlContext, dataView, estimators, initialPipeline);
+        }
+
+        private void CommonCalibratorOnnxConversionTest(IEstimator<ITransformer> calibrator, IEstimator<ITransformer> calibratorNonStandard)
+        {
+            // Initialize variables needed for the ONNX conversion test
+            var (mlContext, dataView, estimators, initialPipeline) = GetEstimatorsForOnnxConversionTests();
+
```

Comment:
Sorry, you might have just introduced a bug here. The mlContext returned from `GetEstimatorsForOnnxConversionTests` is now different from the MLContext used to create the calibrator. Either you should fix `GetEstimatorsForOnnxConversionTests` to use the `MLContext` from the base class or pass it in as a param to this function like I mentioned in my earlier comment.

Reply:
Thank you for the heads up, I have removed the usage of `MLContext` in these tests in lieu of `ML.`